src/App.js

```diff
@@ -1,16 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
 
+let totalLikes = 0;
```

Comment:
`totalLikes` should have been stored in a `useState` variable.

When you update state in React it triggers a re-render automatically.  Updating a global variable won't.  (Also if you instantiated the app twice on the same page you'd be in trouble.)